docs/readthedocs/source/doc/Chronos/Overview/install.md

```diff
@@ -19,7 +19,9 @@
 .. note::
     **Supported Python Version**:
 
-     Chronos only supports Python 3.7.2 ~ latest 3.7.x. We are validating more Python versions.
+     Chronos supports all installation options on Python 3.7.2 ~ latest 3.7.x. For details about different installation options, refer to `here <#install-using-conda>`_.
```

Comment:
One small question here: now chronos don't support 3.7.1？Just feel strange to this 3.7.2

Reply:
> One small question here: now chronos don't support 3.7.1？Just feel strange to this 3.7.2

According to feedback from users before, there exists problems (ray support) with python3.7.0 and python3.7.1.